src/brand/bhyve/boot

```diff
@@ -91,6 +91,7 @@ CDROM_SLOT2     = 7
 DISK_SLOT2      = 8
 PPT_SLOT        = 9
 RNG_SLOT        = 10
+VIRTFS_SLOT     = 11
```

Comment:
name might be confusing if we ever get [virtio-fs,](https://virtio-fs.gitlab.io/) which is different from virtio-9p

Reply:
virtio-9p is known as `VirtFS`. If we ever get virtio-fs, we can either add `virtiofs` as distinct to `virtfs` or just keep one and have a different attribute to select between them. We won't know how it's going to look until it lands in FreeBSD.